### PR TITLE
Fix #346 - Add Apotheosis endless quiver enchantmemt support

### LINKED ISSUE #346: Missing Apotheosis Endless Quiver enchantment support
## Bug Report

**Observed Behaviour**
Looking at #145, I thought that it supports all the enchantments in Apotheosis mod.
But I don't know why, but the support for the endless quiver enchantment doesn't seem to apply.

**Expected Behaviour**
Apply Endless quiver enchantment to my fancy bow and enjoy the combination of tetra & apotheosis

**Minimal setup needed to reproduce**

- Forge version: 1.16.4 - 35.1.36
- Tetra version: Mine is 3.6.0, but I think all version after 2.4.0 has same problem
- Tetra configuration: 
- Other mods: Apotheosis version 4.4.1

**Steps to reproduce**
Creates a single player world and try to apply an endless quiver enchant to the tetra bow.

### PR DESCRIPTION
**Pull request by wogns3623**
Add endless quiver enchantment support
I set the enchantment's magic capacity number to simply double the vanilla infinity enchantment, so change it to your liking!

I would like to edit the Korean translation file for this change, but I can't find it.
How do I edit the Korean translation file?

### CODE REVIEW

**Comment by Cinnamon-Addict** [comment]
This would be nice, although I'm pretty sure that there is a data pack for that, imma check

File: src/main/resources/data/tetra/modules/bow/basic_string.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
         "tetra:enchantment/ma-enchants_floating",
         "tetra:enchantment/ma-enchants_paralysis",
         "tetra:enchantment/apotheosis_life_mending",
+		"tetra:enchantment/apotheosis_true_infinity",
```
Comment (mickelus) [inline src/main/resources/data/tetra/modules/bow/basic_string.json:15]: Could you change this to have the same indentation as the rest of the items in the array?

**Comment by mickelus** [comment]
Heyo, sorry for the slow response. Just started pushing again, could you retarget this towards the `main` branch and fix the comment? 😇

**Comment by wogns3623** [comment]
Thanks for your reply!
I retargeted this to the main branch and added translations for other languages.

**Comment by mickelus** [comment]
Looks like there are still a couple of commits from the 1.15 (master) branch in the MR :/. It's probably easiest if you check out the main branch and get it up to date, then you cherry pick your changes onto that branch!

**Comment by wogns3623** [comment]
@mickelus I create new PR from forked main branch!

**Pull request closed without merging**